src/Forms/HTMLEditor/HTMLEditorField.php

```diff
@@ -231,7 +230,7 @@ private function usesXmlFriendlyField(DataObjectInterface $record): bool
         }
 
         $castingService = CastingService::singleton();
-        $castValue = $castingService->cast($this->Value(), $record, $this->getName());
+        $castValue = $castingService->cast($this->getFormattedValue(), $record, $this->getName());
```

Comment:
This is only called by in `saveInto()` so should probably use `getValue()` instead.

Reply:
Updated